src/main/java/uk/gov/hmcts/darts/common/service/bankholidays/BankHolidaysService.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.darts.common.service.bankholidays;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+@Service
+@RequiredArgsConstructor
+public class BankHolidaysService {
+
+    private final BankHolidaysApi bankHolidaysApi;
+
+    public List<Event> getBankHolidaysFor(int year) {
```

Comment:
What's the reason for passing a year in here? I don't think there are 100's of dates that would be stored, so not sure you're saving much by doing it this way. The requirement for DARTS is going to be along the lines of 'tell me if there is a bank holiday in the next X number of days', so not sure the logic implemented here will actually be used in real life, correct me if i've missed something here?